.gitignore

```diff
@@ -1,4 +1,6 @@
 /node_modules
 /node
+/dist
```

Comment:
I believe we were keeping dist as a src artifact to allow scss files to be deltaed, these are not minified like other projects. 

@pedroportella are you ok with that to stay as it is?

Reply:
yes, all good